### ISSUE TITLE
Styling suggestion for Block components panel

### ISSUE THREAD

**Issue by weavertheme** [WordPress/gutenberg#7063]
**Is your feature request related to a problem? Please describe.**
I'm building a plugin with a block that has more options than might be typical. I've divided the options into several normally closed panels. When a subpanel is opened, it tends to blend into the other options, and gets visually lost.

**Describe the solution you'd like**
I've added this CSS to the editor styling, and it really makes things much visually easier for the user.

```
.components-panel__body.edit-post-block-sidebar__panel.is-opened .components-panel__body.is-opened {
	background-color: rgba(0,0,0,.02);
	border-bottom: 1px solid #888888;
}
```
I think this works pretty well for any of the control panel items, but if you don't consider it appropriate, then it would be nice if the base class of the current block were added around the block components section so this customization rule won't apply to other block's panels.

**Describe alternatives you've considered**
Simply adding stronger top/bottom borders might work as well.

**Comment by chrisvanpatten** [comment]
I agree that the lack of borders between panels in the block inspector makes everything sort of blend together, and really hinders scanability.

**Comment by designsimply** [comment]
For reference, here is a screenshot with the CSS example applied: 

![screen shot 2018-07-03 at tue jul 3 3 44 08 pm](https://user-images.githubusercontent.com/1119271/42246019-4afe68a8-7ed8-11e8-9d4f-7dedaf413738.png)
<sup>Seen at http://alittletestblog.com/wp-admin/post-new.php?gutenberg-demo running WordPress 4.9.6 and Gutenberg 3.1.1 using Chrome 67.0.3396.87 on macOS 10.13.5.</sup>
